core/utils.py

```diff
@@ -91,6 +92,28 @@ class ExplorationConversionError(Exception):
     pass
 
 
+def open_file(
+        filename: str, mode: str, encoding: Union[str, None]='utf-8',
```

Comment:
Shouldn't an encoding always be specified?

Reply:
Sure !!